debug/gdbserver.py

```diff
@@ -1191,6 +1193,12 @@ def test(self):
         time.sleep(1)
         self.gdb.p("buf", fmt="")
 
+        # Disable timer interrupt
+        for hart in self.target.harts:
+            self.gdb.select_hart(hart)
+            self.gdb.interrupt()
+            self.gdb.p("$mie=$mie & ~0x80")
```

Comment:
For more refactoring, can you make this a call to disable_timer() as well?

Probably you need a new `interrupt=False` argument to disable_timer(), and if it's True then it does the interrupt().

Reply:
Your suggestions are very helpful and I have adopted it.